oct/tournament/models.py

```diff
@@ -267,19 +267,22 @@ class TournamentMatch(models.Model):
 
     @property
     def time_str(self):
-        return self.starting_time.strftime("%m/%d %H:%M (%Z)") \
-            if self.starting_time else "Not scheduled"
+        return self.starting_time.isoformat()
```

Comment:
I think this property was for displaying time on the site, which is why strftime was used

Reply:
i should just be able to parse the isoformat string on the frontend side to display the time. honestly i don't think this property would be needed now, since i fixed the datetimefield serialization.

Reply:
yeah the property all together can be removed since datetime values should be automatically serialized as isoformat